src/app/content/content.component.ts

```diff
@@ -9,17 +9,23 @@ import { ArticleListDataService } from "../article-list-data.service";
   styleUrls: ["./content.component.css"]
 })
 export class ContentComponent implements OnInit {
-  articleListData: Article[];
-  filteredArticles: Article[];
+  articleListData: Article[] = [];
+  filteredArticles: Article[] = [];
 
   constructor(
     private updateDataService: UpdateDataService,
     private articleDataSource: ArticleListDataService
```

Comment:
`articleDataSource` should be renamed to `articleListDataService`

Reply:
`Changes Implemented`